.github/workflows/linux.yml

```diff
@@ -17,6 +17,54 @@ on:
       - created
 
 jobs:
+  validate-9_0_1:
+    name: validate.sh ghc-9.0.1
+    runs-on: ubuntu-18.04
+    needs: validate-8_8_3
```

Comment:
9.0.1 job will start after 8.8.3 passes.

Reply:
Aha, thanks. I assumed it would show as "waiting for dependency" or something

Reply:
The *Waiting for status to be reported* jobs are there because they are `required` to pass before non-admins can merge. I.e. they come from repository settings, not in-repo CI config.